### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ gems:
 
 ## Usage
 
-The plugin will automatically generate a RSS feed at `/feed/`. No configuration necessary.
+The plugin will automatically generate an RSS feed at `/feed.xml`. No configuration necessary.
 
 ## Contributing
 
-1. Fork it ( https://github.com/jekyll/jekyll-rss-feed/fork )
```
Comment (XhmikosR) [inline README.md]: BTW, the reason I removed these spaces is that even though (I assume) they were added to match the space the other brackets have, that space is `margin` set via CSS.

**Comment by XhmikosR** [comment]
Added `rss_limit` too.

Do not merge this yet, I want to squash the patches first.

Also, this reminds me, how about having the config option under `rss` like

```yml
rss:
  limit: 10
```

**Comment by benbalter** [comment]
@XhmikosR is there a corresponding pull request to add the RSS limit logic? At first glance I'd be :-1: to add that config setting, let alone three, as outlined over in https://github.com/jekyll/jekyll-rss-feed/issues/5#issuecomment-101393563.

**Comment by XhmikosR** [comment]
@benbalter: what Pull Request?

https://github.com/jekyll/jekyll-rss-feed/blob/master/lib/feed.xml#L15

**Comment by benbalter** [comment]
Ahh. Didn't realize that slipped in there.

**Comment by XhmikosR** [comment]
I tweaked the changes, please have a look now.

**Comment by benbalter** [comment]
Nice. Thanks @XhmikosR!

**Comment by XhmikosR** [comment]
No problem, I was expecting for such a plugin for years since I wanted a common way to generate the feeds apart from using my solution.

**Pull request merged**